### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by eheimbuch**
## Proposed changes

Add issue template

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] PR is well described
- [ ] Target branch is not master (in most cases develop should bet the target of choice) 
- [ ] Code does not conflict with target branch
- [ ] New code is covered with unit tests
- [ ] CHANGELOG.md updated
- [ ] Documentation updated (only necessary for new features or changed behaviour)

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+If you found a bug or want to propose a feature, please describe it as accurately as possible. Use the following checklists. This way we can help you the best. 
+
+### Your checklist for a bug report
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the issue. If you're unsure about any of them, don't hesitate to ask. We're here to help!_
+
+- [ ] Have you described how to reproduce your bug?
+- [ ] Which operating system, browser and versions do you use?
+- [ ] Can you attach some logs or screenshots to this issue?
```
Comment (pfeuffer) [inline issue_template.md:11]: I find check lists with questions irritating. Would it make sense to turn these into demands? Something like

#### Describe how to reproduce the bug
_Please provide a detailled description of what you have to do to reproduce your bug (settings, click paths, ...)_

#### Describe your SCM-Manager setup
_What plugins and what versions do you use? Are you using a server like tomcat? Do you have special settings like proxies? Are you running your instance behind a reverse proxy?_

#### Which operating system, browser and versions do you use?
_Windows, Max or Linux? Chrome, Firefox or IE? What versions?_

#### Log of the bug
_Please mind to check for any sensible information like repository names!_

#### Further information like screenshots

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- [ ] Have you already tried solve it yourself? Can you provide any findings?
```
Comment (pfeuffer) [inline issue_template.md:12]: This is an important point that should move upwards in the first paragraph. Something like

_If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our [mailing list](https://groups.google.com/forum/#!forum/scmmanager) already._

_SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you._

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### Your checklist for a feature request
+
+- [ ] Which problem should this feature solve? 
+- [ ] What is your specific requirement?
+- [ ] Can you implement this feature yourself? We will support you as much as possible.
```
Comment (pfeuffer) [inline issue_template.md:18]: Maybe again concrete headings?

#### Problem to be solved
_Please tell, **what** you want to achieve, not how this should be done._

#### Proposed solution
_Here you can describe your proposed solution._

File: ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
+
+SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you.
+
+### Bug Report
+
+#### Describe how to reproduce the bug
+
+Please provide a detailled description of what you have to do to reproduce your bug (settings, click paths, ...)
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/bug_report.md:16]: detailed

Comment (Pilopa) [reply ISSUE_TEMPLATE/bug_report.md:16]: passive voice is easier to understand when describing generic behaviour: "Please provide a detailed description of what `has to be done` to reproduce the bug (settings, click paths, ...)

File: ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Use this template for reporting bugs.
+---
+
+## Issue description
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
+
+SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you.
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/bug_report.md:10]: `any knowledge of programming` ?

Comment (Pilopa) [reply ISSUE_TEMPLATE/bug_report.md:10]: I dont think you need to add `yourself`. It sounds a bit like telling the user to not bother us and do it himself instead.

File: ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+What plugins and what versions do you use? Are you using a server like tomcat? Do you have special settings like proxies? Are you running your instance behind a reverse proxy?
+
+#### Which operating system, browser and versions do you use?
+
+Windows, Mac or Linux? Chrome, Firefox or IE? What versions?
+
+#### Log of the bug
+
+Please mind to check for any sensible information like repository names!
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/bug_report.md:28]: `remember` instead of `mind`

Comment (Pilopa) [inline ISSUE_TEMPLATE/bug_report.md:28]: `[...] in any logs you provide.`
To make it clear where to check for the sensible information.

File: ISSUE_TEMPLATE/bug_report.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Use this template for reporting bugs.
+---
+
+## Issue description
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/bug_report.md:8]: or `on` our mailing list

File: ISSUE_TEMPLATE/feature_request.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Use this template for requesting features.
+---
+
+## Issue description
+
+If you found a bug or want to have a new feature implemented, please take your time and look whether this issue has been discussed in another issue or our mailing list already.
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/feature_request.md:8]: or `on` our mailing list

File: ISSUE_TEMPLATE/feature_request.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+SCM-Manager is open source. If you have any level of programming skills, maybe you can help yourself. We would love to support you.
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/feature_request.md:10]: any knowledge of programming ?

Comment (Pilopa) [reply ISSUE_TEMPLATE/feature_request.md:10]: I dont think you need to add yourself. It sounds a bit like telling the user to not bother us and do it himself instead.

File: ISSUE_TEMPLATE/feature_request.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+### Feature Request
+
+#### Problem to be solved
+
+Please tell, what you want to achieve, not how this should be done.
```
Comment (Pilopa) [inline ISSUE_TEMPLATE/feature_request.md:16]: I think you can drop the first comma.

**Pull request merged**